app/enterprise/2.1.x/secure-admin-api.md

```diff
@@ -115,6 +119,105 @@ From here, simply apply desired Kong-specific security controls (such as
 [IP restrictions][ip-restriction], or [access control lists][acl]) as you would
 normally to any other Kong API.
 
+If you are using docker to host Kong, you can accomplish a similar task via a declarative configuration such as this one:
+
+``` yaml
+_format_version: "1.1"
+
+services:
+- name: admin-api
+  url: http://127.0.0.1:8001
+  routes:
+    - paths:
+      - /admin-api
+  plugins:
+  - name: key-auth
+
+consumers:
+- username: admin
+  keyauth_credentials:
+  - key: secret
+```
+
+Under this configuration, the Admin API will be available via the `/admin-api` but only for requests accopanied with `?apikey=secret` query
```

Comment:
```suggestion
Under this configuration, the Admin API will be available through the `/admin-api`, but only for requests accompanied with `?apikey=secret` query
```

avoid Latin phrases and abbreviations per style guide

Reply:
Ah, yes I have a tendency to use those. I suspect I have some Ancient Roman ancestry.